Sources/HiPMobile/BusinessLayer/ContentApiFetchers/ExhibitsBaseDataFetcher.cs

```diff
@@ -14,6 +14,7 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
```

Comment:
This looks like an unused using-statement. You should check if it is needed and possibly clean up the imports.